packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1342,15 +1351,19 @@ class RawScrollbar extends StatefulWidget {
   /// {@macro flutter.widgets.Scrollbar.scrollbarOrientation}
   final ScrollbarOrientation? scrollbarOrientation;
 
-  /// Distance from the scrollbar's start and end to the edge of the viewport
-  /// in logical pixels. It affects the amount of available paint area.
+  /// Distance from the scrollbar thumb's start and end to the edge of the
```

Comment:
Distance from the scrollbar thumb's start or end to the nearest edge of the viewport